lib/checkout.dart

```diff
@@ -87,7 +82,8 @@ class _CheckoutViewState extends State<CheckoutView> {
               return Container(
                   child: Column(
                 children: [
-                  GradientButton(
+                  GradientButtonKey(
+                    key: Key("CheckoutButton"),
```

Comment:
I just created this wrapper because only buttons with a `key` is auto instrumented, we need to identify the Buttons name, and `GradientButton` didn't expose the `key` ctor param.